src/index.js

```diff
@@ -1,11 +1,14 @@
+import './index.css';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './components/App';
+import { DataProvider } from './components/ContextData'
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <DataProvider initalState={{}}>
+      <App />
+    </DataProvider>
```

Comment:
You simply wrap the `App` with the `DataProvider` and that serves as the entry point for data into the app. You pull the data into the components individually using either `useData` or `DataConsumer`.